manifests/database/postgresql.pp

```diff
@@ -65,9 +71,9 @@
   }
 
   exec { 'update_pgpass':
-    command => "echo ${database_host}:5432:${database_name}:${database_user}:${database_password} >> /root/.pgpass",
+    command => "echo ${database_host}:5432:${database_name}:${database_user}:${database_password_unsensitive} >> /root/.pgpass",
```

Comment:
Are you not still exposing the password when Puppet is run with `--debug`?

```suggestion
    command => Sensitive("echo ${database_host}:5432:${database_name}:${database_user}:${database_password_unsensitive} >> /root/.pgpass"),
```

(But I don't think I would bother making this conditional on whether `$database_password` is `Sensitive`.)

You also need to do the same with `unless`.

Reply:
(I also wonder why `file_line` hasn't been used here)

Reply:
Correct, didn't think of that. I updated the exec with `Sensitive` for the `command` + `unless`. Maybe this part stems from before `file_line` was even available?